src/backups.py

```diff
@@ -284,11 +316,29 @@ def _generate_backup_list_output(self) -> str:
         backups = json.loads(output)[0]["backup"]
         for backup in backups:
             backup_id, backup_type = self._parse_backup_id(backup["label"])
+            backup_reference = "None"
+            if backup["reference"]:
+                backup_reference, _ = self._parse_backup_id(backup["reference"][-1])
```

Comment:
Because pgbackrest returns a list of references, we pick only the most recent one to represent. 
For example, if an incremental backup `A` depends on the previous differential backup `B`, which depends on a previous full backup `C`, the reference list for `A` will be `[C, B]` --> last one, most recent `B` will be shown